src/backend/chess/app/chess.hoon

```diff
@@ -15,17 +18,18 @@
       position=chess-position
       fen-repetition=(map @t @ud)
       special-draw-available=?
-      ready=?
+      auto-claim-special-draws=?
       sent-draw-offer=?
       got-draw-offer=?
-      auto-claim-special-draws=?
       sent-undo-request=?
       got-undo-request=?
+      opponent=ship
   ==
 +$  state-1
   $:  %1
-      games=(map @dau active-game-state)
-      archive=(map @dau chess-game)
+      games=(map game-id active-game-state)
+      potential-states=(map game-id (list (pair active-game-state card)))
+      archive=(map game-id chess-game)
       challenges-sent=(map ship chess-challenge)
       challenges-received=(map ship chess-challenge)
       rng-state=(map ship chess-commitment)
```

Comment:
What's the thinking behind `potential-states` and `future-states`?

Reply:
If we enable pre-moves (which the Chessground frontend supports), we can allow users to queue up several moves. Advanced players often run into situations where they know exactly what moves the opponent will make and want to queue up moves in advance so as not to waste time off the clock.